src/content/content.js

```diff
@@ -62,11 +62,13 @@ function getFilename() {
   return filename;
 }
 
-function insertEditorFrame() {
+function insertEditorFrame() {	
```

Comment:
The trailing whitespace change is unnecessary (and will create a diff when running `gulp prettify`). Could you please revert this line?